src/test/java/core/basesyntax/ReadWriteTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import core.basesyntax.service.impl.ReadFromFileImpl;
+import core.basesyntax.service.impl.WriteToFileImpl;
+import core.basesyntax.service.interfaces.ReadFromFile;
+import core.basesyntax.service.interfaces.WriteToFile;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReadWriteTest {
```

Comment:
create two classes - the first will check ReadFromFile, the second - WriteToFile

don't use ReadFromFileImpl when checking the result of fileWriter.write() - use Files.readAllLines()
our services should not depend on each other when we are testing a particular service